contracts/ethereum/contracts/UtilityClient.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: GPL
+
+// Copyright 2022 Fluidity Money. All rights reserved. Use of this
+// source code is governed by a GPL-style license that can be found in the
+// LICENSE.md file.
+
+pragma solidity 0.8.16;
+pragma abicoder v2;
+
+import "../interfaces/IFluidClient.sol";
+import "../interfaces/IEmergencyMode.sol";
+import "../interfaces/IERC20.sol";
+
+contract UtilityClient is IFluidClient, IEmergencyMode {
+    IERC20 immutable token_;
+
+    uint immutable deltaWeightNum_;
+    uint immutable deltaWeightDenom_;
+
+    address private oracle_;
+    address private operator_;
+    address private emergencyCouncil_;
+
+    bool private noEmergencyMode_;
+
+    constructor(
+        IERC20 token,
+        uint deltaWeightNum,
+        uint deltaWeightDenom,
+        address oracle,
+        address operator,
+        address council
+    ) {
+        token_ = token;
+        deltaWeightNum_ = deltaWeightNum;
+        deltaWeightDenom_ = deltaWeightDenom;
+
+        oracle_ = oracle;
+        operator_ = operator;
+        emergencyCouncil_ = council;
+        noEmergencyMode_ = true;
+    }
+
+    function drain(address dest) external {
```

Comment:
Have an argument for the "dust collector" in the construction and instead move the money out there, and emit an event when it happens